src/index.ts

```diff
@@ -57,7 +57,18 @@ export interface SopsSyncOptions {
   /**
    * The filepath to the sops file
    */
-  readonly sopsFilePath: string;
+  readonly sopsFilePath?: string;
+
+  /**
+   * If you want to pass the sops file via s3, you can specify the bucket
+   * you can use cfn parameter here
```

Comment:
what do you mean by use a cfn parameter here? I can reference the bucket via cfn parameters?

Reply:
yes - that was the reason behind this PR, We have colleagues, that only synth the templates once and deploy them to different stages, only passing cfn parameters to those templates.